portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,35 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Adedapo's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:powderblue;">
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Adedapo's Portfolio</h1>
       <p>This is my portfolio.</p>
+      <p>Hi, I am Adedapo Adeola. I am an international student currently studying Computer Science and Mathematics.
+          I just finshed my Sophomore year and I will be starting my Junior year by August. My favorite sport is Soccer 
+          and my favorite food is jollof rice.
+      </p>
+
+
+      <p>Here is a link to my LinkedIn profile: <a href = "http://linkedin.com/in/adedapo-adeola-41a901197"> LinkedIn </a></p>
+      <p>Here is a link to my GitHub profile: <a href = "https://github.com/Adedapo1"> GitHub </a></p>  
+      
+      <h2>MY GALLERY</h2>
+        <img src="/images/nigeria.png" /> <br>
+        <img src="/images/soccer.jpg" /> <br>
+        <img src="/images/jollof.jpg" />    
+
+      <h3>A Few Things I Love Photographing Are:</h3>
```

Comment:
optional: if you have photos that you've taken that you could show/to link to, that'd be awesome to see!